fiftyone/utils/sam.py

```
@@ -647,270 +662,317 @@ class SegmentAnythingModel(fout.TorchSamplesMixin, fout.TorchImageModel):
+        for item in batch:
+            for key, val in item.items():
+                if key == "point_labels":
+                    continue
+                elif key == "point_coords":
+                    # Pad point prompts and point labels
+                    point_labels = item["point_labels"]
+
+                    max_points = max([pts.shape[0] for pts in val])
+                    padded_points = []
+                    padded_labels = []
+                    for pts, pts_labels in zip(val, point_labels):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)